src/layers/nodeMarker.ts

```diff
@@ -14,7 +16,16 @@ export default class NodeMarker {
     constructor(options: NodeMarkerOptions) {
         this.options = options;
 
-        const icon = this.getIcon(this.options.color);
+        let icon;
+        if (this.options.type === NodeType.START) {
+            const divIconOptions : L.DivIconOptions = {
+                className: s.nodePin,
+                html: `<div class="${s.pin}" style="--pinColor: ${this.options.color};"/>`,
+            };
+            icon = new L.DivIcon(divIconOptions);
```

Comment:
Instead of L.DivIcon + making marker with pure CSS, using .svg would be better:
https://fontawesome.com/icons/map-marker-alt?style=solid

L.Marker takes an icon as parameter:
https://leafletjs.com/reference-1.3.4.html#marker-option
https://leafletjs.com/reference-1.3.4.html#icon